src/assets/scripts/client/airport/runway/RunwayModel.js

```diff
@@ -119,6 +119,15 @@ export default class RunwayModel extends BaseModel {
          */
         this.queue = [];
 
+        /**
+         * The flight number of the last aircraft that used the runway for takeoff.
+         *
+         * @property lastAircraftTakenoff
+         * @type {string}
+         * @default null
+         */
+        this.lastAircraftTakenoff = null;
+
```

Comment:
Since this is essentially the same as the queue, but in the past, it should follow the same convention. So either we could have the queue store the aircraft themselves (makes the most sense to me but I think @n8rzz had reasons why not to), or have this property also store the aircraft id only. Methods can be added to allow us to retrieve the aircraft from the controller. Resulting names could be `this.lastDepartedAircraftModel` or `this.lastDepartedAircraftId`, based on the path taken.

Reply:
renamed to `lastDepartedAircraftId`. Adding AircraftModel to RunwayModel creates a circular dependency.